DESCRIPTION

```diff
@@ -106,7 +106,10 @@ Suggests:
     knitr,
     testthat,
     tidySingleCellExperiment,
-    ggplot2
+    ggplot2,
+    basilisk,
+    arrow,
+    reticulate
```

Comment:
I have the feeling that if it is for an unexported function, we can put those dependencies in `Suggest`?

Reply:
Ah OK sorry